Sources/View/NibOwnerLoadable.swift

```diff
@@ -41,38 +41,22 @@ public extension NibOwnerLoadable where Self: UIView {
    - returns: A `NibOwnerLoadable`, `UIView` instance
    */
   @discardableResult
```

Comment:
This method doesn't return any result anymore so `@discardableResult` can be removed here